tslint.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should we be using `tslint` or `eslint`? Source: [Dev.to](https://dev.to/benweiser/how-to-set-up-eslint-typescript-prettier-with-create-react-app-3675) - Happy to talk about my implementation of `eslint` w/ typescript, although the post explains it clearly.

Reply:
I have heard tslint will be discontinued soon

Reply:
Yep - That's also mentioned in the post I linked.

Reply:
it will be better to use eslint IMHO as like u guys said it will be deprecated soon

Reply:
tslint is now eslint - let me know what you guys think!